### DISCUSSION TITLE
Issue in cli on sputnik-dao-contract

### DISCUSSION THREAD

**Discussion by GaloisField2718** [near-daos/astro-ui#1301] [Q&A]
Hello everyone, 

I have this script : 
```bash 
#!/usr/bin/env bash

# Put a question and get the answer
prompt_asking () {

	begin=$(date "+%d/%m %T")
	read -p "$hostn : $begin : $1 ?
$hostn : $begin : # "

}

prompt_asking "What account do you want to use "
export NEAR_ACCT=$REPLY

prompt_asking "What DAO do you want to access "
export DAO=$REPLY

prompt_asking "What description do you want to use "
export DESCRIPTION=$REPLY

near call $DAO add_proposal '{
	"proposal" : {
		"description": "'$DESCRIPTION'",
		"kind" : "Vote"
	}
}' --accountId $NEAR_ACCT --amount 0.1 --gas $MAX_GAS
```
This might put a `Vote` in a DAO. 
But, when I put a description as "I want to put a description", I have the error : 
`near call <contractName> <methodName> [args]

schedule smart contract call which can modify state

Options : ...
Unknown Argument :  want, to, put, a, description",
		"kind" : "Vote"
	}
}`
Apparently the description is considered differently as simple text inside proposal. 
An important fact is, without space it works : 
`Scheduling a call: 1901.asso.testnet.add_proposal({
	"proposal" : {
		"description": "HereADescription",
		"kind" : "Vote"
	}
}) with attached 0.1 NEAR
Doing account.functionCall()
Transaction Id HeioMcTSypA3FXCquk4QgYodeczq8ZFP3H9bA2K2Qq5U
To see the transaction in the transaction explorer, please open this url in your browser
https://explorer.testnet.near.org/transactions/HeioMcTSypA3FXCquk4QgYodeczq8ZFP3H9bA2K2Qq5U
18`
So what can I do to solve this issue ?

**Comment by starpause** [comment]
Have you looked at using https://github.com/cloudmex/sputnikdao-cli rather than NEAR CLI directly? 

This seems to be an issue with bash scripting more than it is an issue with NEAR CLI.

**Reply by GaloisField2718** [reply]
Yes I have, but it's less customisable. Yes I think also issue comes from bash scripting but I don't see where is the mistake.
